api/config/routes.rb

```diff
@@ -1,102 +1,91 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-    # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-
-    root :to => 'application#index'
-
     namespace :api do
         namespace :v1 do
-            # instructor routes
-            post '/instructors/delete' => 'instructors#delete'
-            resources :instructors, only: [:index, :create]
+            namespace :admin do
+                # Active User
+                # resource :active_user, only: [:index]
+
+                # Applicants
+                resources :applicants, only: %i[index show create]
+
+                # Application
+                resources :applications, only: :create
+
+                # Assignments
+                resources :assignments, only: %i[show create] do
+                    resources :wage_chunks, controller: :assignment_wage_chunks, only: %i[index create]
+                    resources :offers, path: :active_offer, controller: :active_offers, only: :index do
+                        collection do
+                            post 'create'
```

Comment:
Why is `'create'` a string while the rest are identifiers?

Reply:
`create` has special meaning in Rails and will give you a different route depending on whether it's a symbol vs. string. The `POST` route as a symbol will give you `POST /api/v1/offers/`